save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -94,12 +89,7 @@ class AgentsController(
                     prefix = "save-execution-${execution.id}"
                 )
             }
-                .flatMap { resourcesForExecution ->
-                    val resourcesPath = resourcesForExecution.resolve(TEST_SUITES_DIR_NAME)
-                    execution.downloadTestsTo(resourcesPath)
-                        .then(execution.downloadAdditionalFilesTo(resourcesPath))
-                        .thenReturn(resourcesForExecution)
-                }
+                .subscribeOn(agentService.scheduler)
```

Comment:
`.subscribeOn(agentService.scheduler)`
and then
`.publishOn(agentService.scheduler)`

do we need it?

Reply:
Fixed, only `subscribeOn` is needed